README.adoc

```diff
@@ -656,6 +657,21 @@ foo:
 
 `just` passes the command to be executed as an argument. Many shells will need an additional flag, often `-c`, to make them evaluate the first argument.
 
+==== Windows PowerShell
+
+To be able to run the same justfile on Linux and Windows `set shell` is not enough because it would change the shell on every platform. To set PowerShell as the default shell when running `just` on Windows set `windows-powershell` to true.
+
+```make
+set windows-powershell
+
+hello-windows:
+  Write-Host "Hello, world!"
+
+hello-linux:
+  echo 'Hello, world!'
+```
```

Comment:
I think this example is confusing, because it shows a justfile with a `hello-windows` recipe that won't work on Linux. I would remove it.

```suggestion
```